### PR TITLE
Make FreeRTOS ethernet example work again

### LINKED ISSUE #758: Ethernet support (freertos) broken
Hello, 

@rleh has requested me to investigate what happened to the ethernet support of the freertos implementation in the latest updates of modm (hint, ethernet broke).

looking into the history of the aws (https://github.com/modm-io/modm/commits/develop/ext/aws) I tested the commits one by one ... 

 * commit 1375ff15f569a8f21fa3c24bf57c1e25e2803eac seems to work
 * commit b1ac837c9c2fe24f55b5f8dcf92507667944c11f seems to break it

Ill try looking into this but it seems that this "major update" f'd up a lot.

### PR DESCRIPTION
**Pull request by rleh**
... and add a simple FreeRTOS blink example.

Fixes #758

### CODE REVIEW

File: examples/nucleo_f767zi/freertos_blink/main.cpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+BlinkTask blinkTask;
+
+int
+main()
+{
+	Board::initialize();
+
+	MODM_LOG_INFO << "\n\nReboot: FreeRTOS blink example" << modm::endl;
+
+    modm::rtos::Scheduler::schedule();
```
Comment (salkinium) [inline examples/nucleo_f767zi/freertos_blink/main.cpp]: tabs

Comment (rleh) [reply examples/nucleo_f767zi/freertos_blink/main.cpp]: 🤦🏽

**Review by salkinium — APPROVED** [review]
Thanks!

**Comment by rleh** [comment]
Do we want to trigger a full HAL CI run for this PR, that only touches examples?

**Comment by salkinium** [comment]
> Do we want to trigger a full HAL CI run for this PR, that only touches examples?

I don't think it's necessary.

**Comment by rleh** [comment]
Merge?

**Pull request merged**